### DISCUSSION TITLE
Amasty Subscriptions & Recurring Payments

### DISCUSSION THREAD

**Discussion by carlos-fischer** [breezefront/community#174] [3. Integrations]
Hi,

It would be amazing to have a fast product page + checkout with the ability of having a subscription module with recurring payments. 

This module works for me: Amasty Subscriptions & Recurring Payments, but it slows the buyer process and performance. 
 
https://amasty.com/subscriptions-recurring-payments-for-magento-2.html

**Comment by dimdimich** [comment]
Thank you for your interest. When you say slows does that mean that you can't use Breeze on pages that use Amasty Subscriptions & Recurring Payments?

**Reply by carlos-fischer** [reply]
Yes correct. When I enable Breeze the module doesn't work as well.

**Comment by dimdimich** [comment]
Sorry for delay with response. Can you please first try to follow https://swissuplabs.com/blog/interested-in-amasty-modules-integration-with-breeze-check-this-information/ and contact Amasty and let us know theor response. We had disucssion with Amasty and they said that in case they will have enough requests they will consider adding integration from their side. If they will not do that please let us know and we will add it to our todo list.

**Reply by carlos-fischer** [reply]
Unfortunately, they do not at the moment:
_In regards to compatibility with Breeze theme, unfortunately Subscriptions & Recurring Payments module hasn't been tested for compatibility with it, and our Product team are currently reviewing the possibility of adding such improvement into future module versions. This, however, may take a while due to our current release plans. Therefore, if you require this feature urgently, I'd suggest that you consider the option of custom-coding it for you._

**Comment by dimdimich** [comment]
Thank you for contacting Amasty. So good news is that they already "reviewing the possibility ". Looks like more and more Breeze users are asking them about that. Considering you need this integration as soon as possible please open ticket and our developers will work on that task.
